examples/yacat/yacat.py

```diff
@@ -86,18 +85,18 @@ async def worker_find_password(ctx: WorkContext, tasks):
     # By passing `event_emitter=log_summary()` we enable summary logging.
     # See the documentation of the `yapapi.log` module on how to set
     # the level of detail and format of the logged information.
-    async with Engine(
+    async with Executor(
```

Comment:
I'm not very fond of "Executor" in this context... but of course, homage to the Empire is always appreciated ;) ... 

http://starwarshub.net/wp-content/uploads/2018/10/Executor_BF2.png

Reply:
To be honest, I'm not fond of "executor" either, I really don't like the name. 

But IMO, it does fit in this context very well -- it does precisely entities called "Executor" do in other frameworks that I'm linking to in the PR's description.